sessions/forecasting-concepts.qmd

```diff
@@ -501,24 +501,28 @@ There are many ways to assess calibration and bias but one common way is to use
 
 ### Continuous Case
 
-For a continuous random variable $ X $ with cumulative distribution function (CDF) $ F_X $, the PIT is defined as:
+For a continuous random variable $X$ with cumulative distribution function (CDF) $F_X$, the PIT is defined as:
 
-$ Y = F_X(X) $
+$$
+Y = F_X(X) 
+$$
 
-where $ Y $ is uniformly distributed on $[0, 1]$. 
+where $Y$ is uniformly distributed on $[0, 1]$. 
 
 #### Integer Case
 
-When dealing with integer forecasts, the standard PIT does not yield a uniform distribution even if the forecasts are perfectly calibrated. To address this, a randomized version of the PIT is used. For an integer-valued random variable $ X $ with CDF $ F_X $, the randomized PIT is defined as:
+When dealing with integer forecasts, the standard PIT does not yield a uniform distribution even if the forecasts are perfectly calibrated. To address this, a randomized version of the PIT is used. For an integer-valued random variable $X$ with CDF $F_X$, the randomized PIT is defined as:
 
-$$ U = F_X(k) + v \cdot (F_X(k) - F_X(k-1)) $$
+$$
+U = F_X(k) + v \cdot (F_X(k) - F_X(k-1))
+$$
 
 where:
```

Comment:
```suggestion
where:

```
not 100% sure this will fix it but I still see

<img width="462" alt="image" src="https://github.com/nfidd/nfidd/assets/1156307/0af10a3f-6dd7-412d-b135-4070d9cb6c46">

Reply:
ah damn - might need an indent